assets/src/blocks/ShareButtons/EmailShare/EmailShareFrontend.js

```diff
@@ -0,0 +1,32 @@
+import { SvgIcon } from "../SvgIcon";
+import { parseUtmParams } from '../utils';
+
+const { __ } = wp.i18n;
+
+export const EmailShareFrontend = ({
+  url,
+  utmMedium,
+  utmContent,
+  utmCampaign,
+  gaEvent,
+  gaEventCategory,
+  gaCategory,
+  gaAction,
+  gaLabel,
+  openInNewTab,
+  title,
+  body,
+}) => (
+  <a href={ `mailto:?subject=${title}&body=${body ? encodeURIComponent(body) : ''}\n${url}&${parseUtmParams('email', utmMedium, utmContent, utmCampaign)}` }
+    className="social-share-button email"
+    data-ga-event={gaEvent}
```

Comment:
But the same happens with `event-category`, so I prefer to keep it both and then set a default value. It would be better if the editor prefers to edit all `ga` values.

Reply:
I don't see why that means we need to mirror the implementation of one into the other? I mean, in the same spirit you can create a whole lot of props that are never really used.